app/src/main/java/org/mifos/mobile/ui/fragments/HomeOldFragment.kt

```diff
@@ -271,6 +269,43 @@ class HomeOldFragment : BaseFragment(), HomeOldView, OnRefreshListener {
         }
     }
 
+    /**
+     * Toggles visibility of saving and loan total amounts on home,
+     * and updates the corresponding button icon.
+     */
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        ButterKnife.bind(this, view)
+
+        tv_saving_total_amount.visibility = View.GONE
+        tv_saving_total_amount_hidden.visibility = View.VISIBLE
+        tv_loan_total_amount.visibility = View.GONE
```

Comment:
You can set the initial value of visibility to GONE in the XML file. The same applies to the codes in line number 280 to 283

Reply:
@jawidMuhammadi  I've set the initial visibility of the view using the android visibility attribute in XML.